app/models/card.rb

```diff
@@ -10,13 +10,11 @@ class Card < ApplicationRecord
 		:stripe_card_token, # str
 		:stripe_card_id, # str
 		:stripe_customer_id, # str
-		:holder, :holder_id, :holder_type, # polymorphic cardholder association
-		:inactive # a card is inactive. This is currently only meaningful for nonprofit cards
```

Comment:
Should we delete the inactive column from the cards table? Probably.

Reply:
@wwahammy  I don't understand. Should I keep the inactive column from the cards table for this PR?
Or can I remove the inactive column from the cards table in this PR?

Reply:
Oops, I guess that wasn't clear :smile:   Let's remove the inactive column from the cards table in this PR.

Reply:
The inactive column for the cards table is still in this PR.